### PR TITLE
Feature/white238/rtdmainpage

### PR DESCRIPTION
**Pull request by white238**
https://axom.readthedocs.io/en/feature-white238-rtdmainpage/

### CODE REVIEW

**Comment by white238** [comment]
I am open for suggestions.. I just got sick of clicking on the user guide when i wanted the doxygen

File: src/index.rst [hunk 1]
```diff
@@ -32,64 +32,73 @@ The main drivers of the Axom project are to:
...
+=============
+Documentation
+=============
+
+  * :doc:`Quickstart Guide <docs/sphinx/quickstart_guide/index>`
+  *  `Source documentation <doxygen/html/index.html>`_
+
+.. list-table::
+   :align: center
+
```
Comment (kennyweiss) [inline src/index.rst:58]: Should `core` be here too?
It has doxygen if not sphinx docs.

File: src/index.rst [hunk 2]
```diff
@@ -127,38 +136,21 @@ The project maintains two email lists:
   * 'axom-dev@llnl.gov' is for communication among team members. 
 
 
--------------- 
+---------
 Chat Room
--------------- 
+---------
 
 We also have a chat room on LLNL's Cisco Jabber instance called
```
Comment (kennyweiss) [inline src/index.rst:143]: Should we remove the jabber chat room part and only mention MS Teams?

File: src/index.rst (hunk 1 above)
Comment (rhornung67) [reply src/index.rst:58]: I think I looked yesterday and found no sphinx docs for core.

File: src/index.rst (hunk 2 above)
Comment (rhornung67) [reply src/index.rst:143]: Yes, banish jabber chat to the collaborative software scrapheap!

File: src/index.rst (hunk 1 above)
Comment (agcapps) [reply src/index.rst:58]: I'm willing to leave the link to `core` for later.  They're not quite right in the overall TOC.  I've agreed to write some sphinx docs (ATK-1163), and I propose to fix the doxygen docs and add them here at the same time.

Comment (rhornung67) [reply src/index.rst:58]: Once again, thank you @agcapps ! 🥇

**Pull request merged**